packages/babel-helper-create-class-features-plugin/test/fixtures/plugin-proposal-class-properties/warn-loose-and-assumptions-preset-env/stderr.txt

```diff
@@ -0,0 +1,10 @@
+[proposal-class-properties]: You are using the "loose: true" option and you are explicitly setting a value for the "setPublicClassFields" assumption. The "loose" option can cause incompatibilities with the other class features plugins, so it's recommended that you replace it with the following top-level option:
```

Comment:
Do you think we should also disable this warning?
It's explicitly setting `loose: true`, but in `@babel/preset-env`.

Reply:
Yeah